client-java/README.md

```diff
@@ -0,0 +1,60 @@
+# Palisade Client (Java)
+
+The palisade client provides access to resources.
+
+Example usage:
+
+```
+var client = (JavaClient) Client
+    .create(Map.of(
+        "palisade.client.url", "http://localhost:8081",
+        "palisade.download.threads", "2",
+        "palisade.download.path", "/tmp"));
+
+var futureResult = client
+    .submit(b -> b
+        .userId("user_id")
+        .resourceId("resource_id")
+        .purpose("purpose")
+        .context(Map.of(
+             "key1", "value1",
```

Comment:
Shouldn't these be token and url instead of key1, key2?

Reply:
When creating the request you do not know the token and url. The properties represents just extra info for the server about the request.